src/main/java/ru/yandex/jenkins/plugins/compound/CompoundEnvironmentContributor.java

```diff
@@ -118,7 +118,7 @@ private void buildEnvironmentFor(CompoundSlave slave, EnvVars envs, TaskListener
 					for (InetAddress inetAddress: listedAdresses) {
 
 						// FIXME: maybe should check availability from the ROOT slave, not from master?
-						if (isReachable(inetAddress)) {
+						if (inetAddress != null && !inetAddress.isLinkLocalAddress() && isReachable(inetAddress) ) {
```

Comment:
just follow the trends:)
https://www.jetbrains.com/idea/help/reformatting-source-code.html